### PR TITLE
feat(v2 SDK): Update v2 SDK with new v2 API definition

### PR DESCRIPTION
**Pull request by AsnaNesry**
## PR summary
feat(v2 SDK): Update v2 SDK with new v2 API definition

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?  
 v2 SDK 
## What is the new behavior?  
Updated  v2 SDK 

## Does this PR introduce a breaking change?    
- [x] Yes
- [ ] No

### CODE REVIEW

File: modules/watsonx-data-v2/pom.xml [hunk 1]
```diff
@@ -13,15 +13,18 @@
...
+        <maven.javadoc.skip>true</maven.javadoc.skip>
+    </properties>
+
     <dependencies>
+        <!-- There should be a dependency for each module whose request example class exists in this "examples" module. Add 
+            new "dependency" entries here as needed when you add a request examples class for a new service. Note: the "artifactId" values 
+            below should reflect the "artifactId" value specified in each module's pom.xml file (i.e. not the module's directory name). -->
         <dependency>
             <groupId>com.ibm.cloud</groupId>
-            <artifactId>sdk-core</artifactId>
```
Comment (fanfeigithub) [inline modules/watsonx-data-v2/pom.xml:19]: Why need to change this file?

Comment (AsnaNesry) [reply modules/watsonx-data-v2/pom.xml]: @fanfeigithub i reverted that change. Could you please review it again?

**Review by emilyz-github — APPROVED** [review]
lgtm

**Pull request merged**